.github/workflows/functional.yml

```diff
@@ -0,0 +1,109 @@
+name: Build Images
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      diff_branch:
+        description: 'Upstream branch for determining changes'
+        default: 'master'
+      run_all:
+        description: 'Run all tests regardless of changes?'
+        default: 'false'
+env:
+  IMAGE_REGISTRY: quay.io
+  IMAGE_USER: rydrew
```

Comment:
maybe, replace it with redhat_perf_ci account and store credentials in the repository secrets.

Reply:
Yep for sure. There are a couple of changes that need to be made to transition from my test accounts over to the more generic ones we would use for cloud-bulldozer.